applications/GeoMechanicsApplication/custom_elements/transient_Pw_element.cpp

```diff
@@ -567,58 +568,58 @@ void TransientPwElement<TDim, TNumNodes>::CalculateAndAddRHS(VectorType&       r
                                                              ElementVariables& rVariables,
                                                              unsigned int      GPoint)
 {
-    KRATOS_TRY;
+    KRATOS_TRY
 
     this->CalculateAndAddCompressibilityFlow(rRightHandSideVector, rVariables);
     this->CalculateAndAddPermeabilityFlow(rRightHandSideVector, rVariables);
     this->CalculateAndAddFluidBodyFlow(rRightHandSideVector, rVariables);
 
-    KRATOS_CATCH("");
+    KRATOS_CATCH("")
 }
 
 //----------------------------------------------------------------------------------------
 template <unsigned int TDim, unsigned int TNumNodes>
 void TransientPwElement<TDim, TNumNodes>::CalculateAndAddPermeabilityFlow(VectorType& rRightHandSideVector,
                                                                           ElementVariables& rVariables)
 {
-    KRATOS_TRY;
+    KRATOS_TRY
 
-    this->CalculatePermeabilityFlow(rVariables.PPMatrix, rVariables.PVector, rVariables);
+    auto permeability_flow = this->CalculatePermeabilityFlow(rVariables);
 
     // Distribute permeability block vector into elemental vector
-    rRightHandSideVector += rVariables.PVector;
+    rRightHandSideVector += permeability_flow;
```

Comment:
Looking again, why don't we write;
rRightHandSideVector += this->CalculatePermeabilityFlow(rVariables);

and similar in other places.

Reply:
done